src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -2157,19 +2157,10 @@ subroutine PhotosynthesisTotal (fn, filterp, &
             endif
             if ( use_c14 ) then
 
-               ! determine latitute sector for radiocarbon bomb spike inputs
```

Comment:
This is moved to go inside of the C14_Bombspike subroutine which isolates knowledge of the grid to the place where it's read.